internal/driver/config/config.schema.json

```
@@ -324,7 +324,10 @@
           "type": "string",
           "description": "Set this to the tracing backend you wish to use. Currently supports jaeger. If omitted or empty, tracing will be disabled.",
```

```suggestion
          "description": "Set this to the tracing backend you wish to use. If omitted or empty, tracing will be disabled.",
```
Don't the others need to have some config as well? Also I noticed somewhere in the tracing section we have Hydra still instead of Keto